### ISSUE TITLE
Research: track Fares v2 adoption

### ISSUE THREAD

**Issue by evansiroky** [cal-itp/data-infra#473]
## Question

It would be nice to have an auto-generated report showing the progress of fares v2 adoption in feeds.

## Metrics

* A pie chart showing Fares v2 adoption in each feed stratified into 3 categories:
  * Complete
  * In progress
  * Missing

## Data sources

* Private Google Sheet (message @evansiroky)? Feeds themselves?

## (Data Servicess Team to Copy and Fill Out Below)

The QuVR MD template below will be filled out by a member of the data services team.
This allows us to describe the request, in a way that is easy to hand-off for analysis.
After the research phase, we will sync with the asker to figure out if the metric and dashboard pieces are needed.

Before starting research:

* **Question**: Question written as a single sentence.
* **View**: E.g. `views.gtfs_schedule_fact_daily_feed_files`.
* **Research**:
  - How should the results be presented?
  - When are they needed by?

After reviewing research with the asker:

* **Metric**: what specific calculations are needed?
* **Dashboard**: where should we put the result?
